packages/frontend/scripts/config-react-env.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+require("dotenv/config");
+
+const SUPPORTED_BUILD_MODES = ["production", "staging", "library"];
+
+const buildMode = process.argv[2];
+if (!buildMode) {
+    console.error("A build mode parameter is required");
+    process.exit(1);
+}
+if (SUPPORTED_BUILD_MODES.indexOf(buildMode) < 0) {
+    console.error(
+        `Unsupported build mode ${buildMode}. Supported values are:\n    ${SUPPORTED_BUILD_MODES.join(
+            ", "
+        )}`
+    );
+    process.exit(1);
+}
+
+const requireEnv = (envName) => {
+    const env = process.env[envName];
+    if (!env) throw new Error(`a ${envName} env is required`);
+};
+
+const isEnvDefined = (envName) => !!process.env[envName];
```

Comment:
I think we can avoid defining this function since it's only used once.

Reply:
Addressed in 9bf473c9156b237eecc3cb427014faf2ce96eab4